build.py

```diff
@@ -249,6 +249,8 @@ def unpack(self):
                 zf.extractall(path=destdir)
         # .. and set the builder object to point to the file
         self.builder.ninja = destfile

```

Comment:
Do we actually need self.builder.ninja? if path is enough I guess I'd just remove the other one